package.json

```diff
@@ -43,6 +41,5 @@
     "commitizen": {
       "path": "cz-conventional-changelog"
     }
-  },
-  "version": "1.0.1"
```

Comment:
Just confirming, we dont need the version field anymore? This is handled by semantic release?